### ISSUE TITLE
Error Encountered: `Undefined array key "argv"` during plugin check in version 0.2.2

### ISSUE THREAD

**Issue by mukeshpanchal27** [10up/plugin-check#338]
When anyone check plugin using official WP.ORG plugin they get below error:
```
Warning:  Undefined array key "argv" in /s4-concentratesister/wordpress/wp-content/plugins/plugin-check/inc/class-phpcs-runner.php on line 126
```

Props to @mujuonly repoting.

**Comment by afragen** [comment]
I can confirm.

To fix line 126 just needs a guard of sort.

I changed the line from
```php
$orig_cmd_args = $_SERVER['argv'];
```

to 
```php
$orig_cmd_args = $_SERVER['argv'] ?? [];
```

**Comment by afragen** [comment]
I have no real idea how this plugin is built and it makes it hard to create a PR.

**Comment by mukeshpanchal27** [comment]
@afragen, you can open a PR against the `legacy-plugin` branch as **Plugin Check** plugin manage through that branch. https://github.com/WordPress/plugin-check/tree/legacy-plugin

**Comment by afragen** [comment]
Thanks @mukeshpanchal27

**Comment by mukeshpanchal27** [comment]
Closed #340
